editor/editor_inspector.cpp

```diff
@@ -2678,11 +2678,13 @@ void EditorInspector::update_tree() {
 
 		// Get the property label's string.
 		String name_override = (path.contains("/")) ? path.substr(path.rfind("/") + 1) : path;
+		String dot_char;
 		String feature_tag;
 		{
-			const int dot = name_override.find(".");
+			const int dot = name_override.rfind(".");
 			if (dot != -1) {
-				feature_tag = name_override.right(dot);
+				dot_char = ".";
+				feature_tag = name_override.substr(dot + 1, -1);
```

Comment:
This should not be changed to `rfind`, as other places like ProjectSettings also use the _first_ dot as separator. It could be another PR if you want to change this behavior.

The `dot_char` is redundant here. Using `feature_tag = name_override.substr(dot);` is OK I think. `feature_tag` is here only to be concatenated back to the property path.

Reply:
Delimiters are still needed, spaces may be more appropriate, otherwise the two words may be merged into one word.

Reply:
I mean, you stripped the dot from `feature_tag` by using `dot + 1` instead of `dot`, and used a dedicated `dot_char` to store it.

If you use `substr(dot)`, then `dot_char` won't be needed.